ee/vellum_ee/workflows/display/nodes/base_node_display.py

```diff
@@ -205,7 +206,7 @@ def serialize(self, display_context: "WorkflowDisplayContext", **kwargs: Any) ->
             "id": str(node_id),
             "label": self.label,
             "type": "GENERIC",
-            **self.serialize_generic_fields(display_context),
+            **self.serialize_generic_fields(display_context, exclude=["outputs"]),
             "adornments": adornments,
             "attributes": attributes,
```

Comment:
I would move these two above `serialize_generic_fields` to minimize the diff in `node-definitions.json`